variants/aws-ecs-1-nvidia/Cargo.toml

```diff
@@ -20,6 +20,7 @@ included-packages = [
 # core
     "release",
     "kernel-5.10",
+    "netdog-wicked",
```

Comment:
checking my understanding, when a variant directly includes this subpackage, it overwrites the link created by `netdog-systemd-networkd`, right? are we sure that the packages are always installed in the correct order?

Reply:
Yes essentially, so netdog defines two subpackages:

netdog-systemd-networkd which provides netdog:1 (epoch 1)
netdog-wicked which provides netdog:0 (epoch 0)

So way to think of the above without it the pseudo dnf install resolution would go dnf install netdog (find the default latest epoch) this would auto resolve to netdog-systemd-networkd since it provides netdog with highest epoch

However if you manually say install netdog-wicked it will ensure that netdog is provided by netdog:0 or netdog-wicked

Reply:
got it, thanks for the explanation